modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -116,7 +116,7 @@ namespace {{packageName}}.Client
 
         // Creates and sets up a RestRequest prior to a call.
         private RestRequest PrepareRequest(
-            String path, {{^netStandard}}RestSharp.{{/netStandard}}Method method, Dictionary<String, String> queryParams, Object postBody,
+            String path, RestSharp.Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
```

Comment:
@jimschubert yes, you're right. I'll take note of that. Should the {{netStandard}} tags be removed?